test/adagrams.test.js

```diff
@@ -1,9 +1,4 @@
-import {
-  drawLetters,
-  usesAvailableLetters,
-  scoreWord,
-  highestScoreFrom,
-} from "adagrams";
+import A from "adagrams";
```

Comment:
Nice working getting some practice with writing a JS class. I like that you used the `static` keyword for the methods.